go-packages/meep-model/validator.go

```diff
@@ -70,6 +70,14 @@ func ValidateScenario(jsonScenario []byte) (validJsonScenario []byte, status str
 		log.Error(err.Error())
 		return nil, ValidatorStatusError, err
 	}
+
+	if name != "" {
+		if scenario.Name != name {
+                        err = errors.New("Scenario creation name " + name + " incompatible with scenario body content name " + scenario.Name + ". They must be the same.")
+                        return nil, ValidatorStatusError, err
+		}
+	}
+
```

Comment:
This code should be in the private _validateScenario_ function where the scenario format is verified.

Reply:
After discussions, leaving it as is. Versions is already checked at that level and using the private validateScenario function more as a validate deployment. Checks in the private function is limited to name and id uniqueness.